src/control/common/file_utils.go

```diff
@@ -257,3 +258,74 @@ func FindBinary(binName string) (string, error) {
 
 	return adjPath, nil
 }
+
+// Copy file from src to dst.
```

Comment:
These comments should start with the name of the function being documented, and be in the form of a complete sentence. For example something like "CpFile copies a file from src to dst." The comments are used to autogenerate documentation.